### PR TITLE
Push notifications + misc fixes

### PR DESCRIPTION
**Pull request by vlad-x**

### CODE REVIEW

File: app/src/utils/push.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      'Content-type': 'application/json',
+      Authorization: `Bearer ${token}`,
+    },
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok')
+      }
+      return response.json()
+    })
```
Comment (eugeek) [inline app/src/utils/push.ts:37]: No needed here

File: backend/src/push.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import webPush from "web-push";
+import assert from "node:assert";
+
+import { SAVE_PUSH_TOKEN, GET_PUSH_TOKENS } from "./database/queries.js";
+import { allQuery } from "./database/index.js";
```
Comment (eugeek) [inline backend/src/push.js:5]: missed import `runQuery`

**Comment by vlad-x** [comment]
@eugeek thanks for the feedback, it should be fixed now! 🙌

**Comment by eugeek** [comment]
LGTM

**Pull request merged**